gravitee-apim-perf/grafana/docker-compose.yml

```
@@ -0,0 +1,59 @@
+version: '3.4'
+
+services:
+  db:
```

just to improve semantic
```suggestion
  promscale_db:
```